<launch>
  <node pkg="robot_base" type="odom.py" name="robot_base_odom">
    <param name="topic" value="/robot_base/odometry"/>_
  </node>

  <node pkg="robot_base" type="utils.py" name="robot_base_utils">
    <param name="enable_clear_costmap" value="True"/>
    <param name="enable_amcl_update" value="True"/>
    <param name="clear_costmap_period" value="10"/>
    <param name="amcl_updates" value="10"/>
  </node>
</launch>